src/godirect.js

```diff
@@ -1,20 +1,27 @@
 import Device from './Device.js';
 import WebBluetoothDeviceAdapter from './WebBluetoothDeviceAdapter.js';
+import WebUsbDeviceAdapter from './WebUsbDeviceAdapter.js';
 
 const godirect = {
   /**
   * This returns a Device instance
   * @name createDevice
-  * @param {BluetoothDevice} bleDevice
+  * @param {Device} nativeDevice - this will be either a BluetoothDevice or HIDDevice depending on how it was created
   * @param {config} config
   * @returns {Promise} Promise object represents a Device instance
   */
-  async createDevice(bleDevice, { open = true, startMeasurements = true } = {}) {
-    let adapter = bleDevice;
+  async createDevice(nativeDevice, { open = true, startMeasurements = true } = {}) {
+    let adapter = nativeDevice;
 
-    // If not a go direct adapter, assume a web bluetooth device
+    // If not a go direct adapter then create it based on what was passed in
     if (!adapter.godirectAdapter) {
-      adapter = new WebBluetoothDeviceAdapter(bleDevice);
+      if (nativeDevice.gatt) {
+        adapter = new WebBluetoothDeviceAdapter(nativeDevice);
+      } else if (nativeDevice.collections[0].outputReports[0]) {
```

Comment:
you may want to ensure that nativeDevice.collections exists, but I'm not sure if you expect that it always will: 
`nativeDevice.collections && nativeDevice.collections[0].outputReports[0]`